src/sonic-config-engine/setup.py

```diff
@@ -50,6 +55,14 @@
         'sonic_yang_cfg_generator'
     ]
 
+for package in sonic_dependencies:
+    try:
+        pkg_resources.get_distribution(package)
+    except pkg_resources.DistributionNotFound:
+        print(package+" is not found!")
```

Comment:
>print [](http://example.com/codeflow?start=8&length=5)

Print 2 error messages to stderr.

Reply:
It should be used by python2 and python3.
we can't use print. Use sys.stderr.write instead.

Reply:
you can use print(msg, file=sys.stderr)

Reply:
python2 doesn't support it.